bundles/http_admin/http_admin/src/activator.c

```diff
@@ -61,6 +61,15 @@ static int http_admin_start(http_admin_activator_t *act, celix_bundle_context_t
     long prop_port_max = celix_bundleContext_getPropertyAsLong(ctx, HTTP_ADMIN_PORT_RANGE_MAX_KEY, HTTP_ADMIN_PORT_RANGE_MAX_DFT);
     long num_threads = celix_bundleContext_getPropertyAsLong(ctx, HTTP_ADMIN_NUM_THREADS_KEY, HTTP_ADMIN_NUM_THREADS_DFT);
 
+    const char* listPortEnv = getenv(HTTP_ADMIN_LISTENING_PORTS_KEY);
```

Comment:
please use the bundlecontext to get the property. If not found in the properties, it will check env as well.

Reply:
But then you would not be able to override the bundle properties with an env var?

Reply:
Removed, a few lines up was already using bundle context.